setup.py

```diff
@@ -32,9 +32,10 @@ def run(self):
     'Click>=6.0',
     'docker>=3.5',
     'python-dotenv',
-    'requests',
+    'requests<2.21',
     'fstrings',
     'msrestazure~=0.4.32',
+    'azure-cli-core<2.0.55',
```

Comment:
same as above

Reply:
`azure-cli-core` depends on a beta version of PyYAML starting version 2.0.55, which will conflict with docker-compose. See https://github.com/Azure/iotedgedev/issues/373

Reply:
https://github.com/Azure/azure-cli/blame/dev/src/azure-cli-core/setup.py#L71

Reply:
Lower bound specified.